rush.json

```diff
@@ -92,7 +92,7 @@
    * Specify a SemVer range to ensure developers use a NodeJS version that is appropriate
    * for your repo.
    */
-  "nodeSupportedVersionRange": ">=8.0.0 <10.0.0",
+  "nodeSupportedVersionRange": ">=8.0.0 <9.0.0 || >=10.0.0 <11.0.0",
```

Comment:
For node 10, maybe restrict it to more recent versions...

Reply:
what do you mean?

Reply:
or... do you know of problems with some versions?

Reply:
What do you think a good start version would be? I'm not sure how to determine that.

Reply:
gotcha; maybe just start at the current LTS then? (10.14.1) as we're just starting to test now.

There were a few security updates recommending minbar 10.14.0, and the .1 patch was specifcally for windows msi issues.

https://nodejs.org/en/blog/vulnerability/november-2018-security-releases/

Reply:
> What do you think a good start version would be? I'm not sure how to determine that.

(1) an LTS version, and (2) something you've actually tested yourself.

It is super frustrating to waste an hour investigating somebody's issue, only to find out that it was just a buggy build of NodeJS.  We got burned by this repeatedly until we finally started insisting on LTS versions.

Reply:
Recommended DL https://groups.google.com/forum/#!forum/nodejs-sec